### PR TITLE
#167456659 redirect endpoints response to frontend

### PR DESCRIPTION
**Pull request by Eazybee**
#### What does this PR do?
This PR redirect endpoints response to frontend

#### Description of Task to be completed?
- redirect oauth endpoint to frontend
- redirect resetPassword endpoint to frontend
- validate and handle response of breaking endpoints
- refactor code base

#### How should this be manually tested?
- Run ```npm run test```

#### Any background context you want to provide?
- You need `node` and `npm` installed 
- Run ```npm install``` to install dependencies 
- You need to install an editorconfig extension in your vscode

#### What are the relevant pivotal tracker stories?
[#167456659](https://www.pivotaltracker.com/story/show/167456659)

### CODE REVIEW

File: server/middlewares/existingUserChecker.js [hunk 1]
```diff
@@ -4,28 +4,22 @@ import model from '<serverModels>';
...
-    message = 'Username exist';
   }
   if (message) {
     const err = new Error(message);
     return displayError(err, res, 409);
   }
+
   req.formattedValues = formattedValues;
-  next();
+  return next();
```
Comment (benfluleck) [inline server/middlewares/existingUserChecker.js:23]: Why is this here?

Comment (Eazybee) [reply server/middlewares/existingUserChecker.js:23]: eslint rule consistence return

https://eslint.org/docs/rules/consistent-return

**Review by benfluleck — APPROVED** [review]
